app/src/main/java/io/nekohasekai/sagernet/vpn/DashboardActivity.kt

```diff
@@ -580,14 +573,18 @@ class DashboardActivity : BaseThemeActivity(),
         debugLog("User_earned_the_reward")
     }
 
-    override fun onPingTestFinished() {
+    private fun updateUIAfterPingTest() {
         runOnUiThread {
             binding.clpbPing.visibility = View.GONE
             binding.clIconPing.visibility = View.VISIBLE
             AppRepository.refreshServersListView()
         }
     }
 
+    override fun onPingTestFinished() {
+        updateUIAfterPingTest()
```

Comment:
This name is very generic! Change the name to `resetPingBtnUI()`.
It should exactly explains what it does!